lib/versioning/rez/index.spec.ts

```diff
@@ -0,0 +1,666 @@
+import type { RangeStrategy } from '../../types';
+import semver from '../semver';
+import { api as versioning } from '.';
+
+describe('versioning/rez/index', () => {
+  describe('equals', () => {
+    it.each([
+      ['1', '1'],
+      ['1.0', '1'],
+      ['1.0.0', '1'],
+      ['1.9.0', '1.9'],
+    ])('%s == %s', (a, b) => {
+      expect(versioning.equals(a, b)).toBe(true);
+    });
```

Comment:
```ts
test.each`
  a             | b        | expected
  ${'1'}        | ${'1'}   | ${true}
  ${'1.0'}      | ${'1'}   | ${true}
  ${'1.0.0'}    | ${'1'}   | ${true}
  ${'1.9.0'}    | ${'1.9'} | ${true}
  ${'1'}        | ${'2'}   | ${false}
  ${'1.9.1'}    | ${'1.9'} | ${false}
  ${'1.9-beta'} | ${'1.9'} | ${false}
`('equals("$a", "$b") === $expected', ({ a, b, expected }) => {
  expect(versioning.equals(a, b)).toBe(expected);
});
```

Reply:
For more examples, take a look at `cargo`...`hex` versioning schemes that already are refactored this way

Reply:
Btw, you don't have to align the columns manually, Prettier will help you

Reply:
Updated. Please have another look.